autorest/azure/environments.go

```diff
@@ -150,7 +154,8 @@ var (
 		ServiceBusEndpointSuffix:     "servicebus.cloudapi.de",
 		ServiceManagementVMDNSSuffix: "azurecloudapp.de",
 		ResourceManagerVMDNSSuffix:   "cloudapp.microsoftazure.de",
-		ContainerRegistryDNSSuffix:   "azurecr.io",
+		ContainerRegistryDNSSuffix:   "",
```

Comment:
This is supposed to be empty?

Reply:
beat me to it, there is no endpoint in [German Cloud]( https://azure.microsoft.com/en-us/global-infrastructure/services/?regions=china-non-regional,china-east,china-east-2,china-north,china-north-2,germany-non-regional,germany-central,germany-northeast,us-central,us-east,us-east-2,us-north-central,us-south-central,us-west-central,us-west,us-west-2&products=all )  ( also searched internally )

Reply:
So if we omit it entirely and somebody tried to use this for German configuration will they get a compile time error?  I would prefer that to possible weird behavior due to empty string.  Or should the value be the one for public?

Reply:
removed per our discussion